src/AuthenticationResponse.js

```diff
@@ -443,17 +443,26 @@ class AuthenticationResponse {
     let rp = response.rp
     let provider = rp.provider
     let decoded = response.decoded
+    let isFreshJwks = false
 
     return Promise.resolve(provider.jwks)
 
-      .then(jwks => jwks ? jwks : rp.jwks())
+      .then(jwks => jwks ? jwks : (isFreshJwks = true, rp.jwks()))
 
       .then(jwks => {
-        if (decoded.resolveKeys(jwks)) {
+        if (decoded.resolveKeys(jwks))
```

Comment:
The Standard.js code style requires curly braces, even for one-line if/then statements. Please re-add.